server/routes/overview/overviewController.ts

```diff
@@ -24,11 +21,7 @@ export default class OverviewController {
       const allFunctionalSkills = await this.curiousService.getPrisonerFunctionalSkills(prisonNumber, req.user.username)
       const functionalSkills = mostRecentFunctionalSkills(allFunctionalSkills)
 
-      const goals = await this.educationAndWorkPlanService.getGoalsByStatus(
-        prisonNumber,
-        GoalStatusValue.ACTIVE,
-        req.user.username,
-      )
+      const { goals } = res.locals.goals || { goals: [], problemRetrievingData: true }
```

Comment:
I think this should just be
```
const { goals } = res.locals
```

IE. use the whole goals object from res.locals; as we shouldnt need to worry about null checks or similar as its a fair assumption that your new middleware will have run first 👍